### DISCUSSION TITLE
Downlowad report

### DISCUSSION THREAD

**Discussion by FelipeClement** [Nesvilab/FragPipe-Analyst-doc#37] [General]
Hi there! Here is Felipe from Spain, working at UCM proteomics facility. We are starting with FragPipe and FragPipe-Analyst with timsTOF data. We have run some LFQ experiments and analyzed them is FragPipe. All (almost) works properly but we got some issues in FragPipe-Analyst, were some features did not work, like Sample Correlation chart and pathway enrichment/gene ontology. This is not a worry for us but Download Report works only sometimes and this is a little upset cause we can collect there all the figures at a time instead of ussing the snipping tool all the time.
Leaving this aside, FrapPipe seems to be the most powerfull and fast workflow to deal with proteomics data, thank you very much to developers and contributors.

Felipe

**Comment by hsiaoyi0504** [comment]
@FelipeClement Could you give us some more hints about reproducing the issue? It's nearly possible to fix things without further details.

**Reply by FelipeClement** [reply]
Thanks for your quick answer, but there are no more hints available. when you click the Download report button, nothing happens.
I am using the last version of Mozilla Firefox as browser, and I tried it with last version of google chrome: the same behavior

Felipe

**Reply by hsiaoyi0504** [reply]
@FelipeClement Could you share your input file with me? yihsiao@umich.edu. Again, it's not possible to guess everything from those info.

**Comment by hsiaoyi0504** [comment]
Closed since no response
